controls/V-81883.rb

```diff
@@ -55,7 +55,13 @@
   tag "documentable": false
   tag "severity_override_guidance": false
 
-  describe yaml(input('mongod_conf')) do
-    its(%w{security enableEncryption}) { should cmp 'true' }
+  describe.one do
```

Comment:
@HackerShark this is not what we discussed.
first half of the or condition is good ...second half should say `{ should match /--enableEncryption true/}`

and an additional condition outside 
```
    describe processes('mongod') do
      its('commands.join') { should_not match /--enableEncryption false/}
    end
```